int32 bboxCounts

float64[100] xMini
float64[100] yMini
float64[100] zMini

float64[100] xMaxi
float64[100] yMaxi
float64[100] zMaxi